simapp/app.go

```diff
@@ -427,6 +428,20 @@ func (app *SimApp) SimulationManager() *module.SimulationManager {
 	return app.sm
 }
 
+func consensusParamsKeyTable() params.KeyTable {
```

Comment:
Should this be somewhere public outside of simapp so it's re-usable?

Reply:
It was originally made public, but @fedekunze suggested to make it private. If it is public, it does not belong in `simapp` and idk of a better place for it to exist. Maybe different apps will want different verification functions?

Reply:
Sounds like the same use case as `codec/std`. It's not really codec related per se, but l think it makes sense there. Maybe we rename that package just `std`?

Reply:
I'd opt to just leave this as-is. 

I'd advise against `std/` becoming a hodgepodge of stuff, like `types/` is now, which is an anti-pattern.

Reply:
But shouldn't the SDK include the out of box components to spin up a zone without needing to custom implement stuff like this? It seems like most zones would want to just have something like this available to use without needing to just copy and paste. And I would argue that `std/` actually is the place to put this stuff. `std/` to me implies the out-of-box default stuff that is needed to build a simple cosmos sdk zone, but that can be overidden if needed.

Reply:
Also, I would argue that rather than being a hodgepodge we could have conventions for how `types/` and `std/` are used.

The criteria I have in mind are:
* `types/` is the non-module stuff that all modules can depend on.
* `std/` is the inter-module stuff in the SDK that zones can depend on as a starting point, but override

Beyond just the scope of this small bit of code, I would love to come to some architectural alignment on how we make these sorts of decisions.

Reply:
> But shouldn't the SDK include the out of box components to spin up a zone without needing to custom implement stuff like this?

Yes. And that is what the SDK attempts to provide. Here `consensusParamsKeyTable` can be used by other apps or not. So I agree we can make it public.

> And I would argue that std/ actually is the place to put this stuff. std/ to me implies the out-of-box default stuff that is needed to build a simple cosmos sdk zone, but that can be overidden if needed.

I agree. We should consider creating a root-level package: `std`. Moving types there.

> Also, I would argue that rather than being a hodgepodge we could have conventions for how types/ and std/ are used.

I agree, with one caveat. IMHO, `types/` shouldn't even exist. Rather, we create smaller packages with smaller and specific scope (to rid ourselves of this anti-pattern).

Reply:
I created a `/std` pkg with a `doc.go`.